src/main/java/uk/ac/ebi/eva/evaseqcol/controller/seqcol/SeqColController.java

```diff
@@ -83,6 +83,13 @@ public ResponseEntity<?> getSeqColByDigestAndLevel(
         return new ResponseEntity<>(HttpStatus.NOT_FOUND);
     }
 
+    @GetMapping(value = "/collection/{digest}/metadata")
```

Comment:
I understand the rational for making the metadata available for a given digest and we could keep it for now.
However I think the metadata should injected in the SeqCol level2 to and be added with their own properties.
```
{
  "sequences": [...],
  "names": : [...],
  "length": : [...],
  "naming_convention": GENBANK,
  "source_id": "GCA_000001",
  "source_url": "https://..."
  "ingested_on": 2024-04-21T12:00:00
}
```

This could be added to `SeqColService.getSeqColByDigestAndLevel`

Reply:
> Could also be added on request via a path variable - e.g. `/collection/{digest}?level=2&metadata=true` - in case anyone wants the "plain" level 2 object?

I think that's a very good idea, it preserves the endpoint specified/required by the spec, and extend it with other optional parameters.